deploy/eraser.yaml

```diff
@@ -453,6 +453,7 @@ data:
               - HIGH
               - MEDIUM
               - LOW
+            ignoredStatuses:
```

Comment:
you can revert any changes to here. manifest_staging will automatically be promoted when we cut a new release

Reply:
Removed my changes here thanks! 👍🏼